webpack.config.js

```diff
@@ -262,7 +262,6 @@ if (NODE_ENV === "hot") {
 
   config.plugins.unshift(
     new webpack.NoEmitOnErrorsPlugin(),
-    new webpack.NamedModulesPlugin(),
```

Comment:
# Remove `NamedModulesPlugin` (1746fd229186a2739c352fe98b8a0a345c191d7c)
```
[frontend] [webpack-cli] TypeError: compiler.plugin is not a function
[frontend]     at UnusedFilesWebpackPlugin.apply (/Users/offbynone/code/work/metabase/node_modules/unused-files-webpack-plugin/lib/index__8.0.0__.js:105:14)
[frontend]     at createCompiler (/Users/offbynone/code/work/metabase/node_modules/webpack/lib/webpack.js:74:12)
[frontend]     at create (/Users/offbynone/code/work/metabase/node_modules/webpack/lib/webpack.js:123:16)
[frontend]     at webpack (/Users/offbynone/code/work/metabase/node_modules/webpack/lib/webpack.js:147:32)
[frontend]     at WebpackCLI.f [as webpack] (/Users/offbynone/code/work/metabase/node_modules/webpack/lib/index.js:54:15)
[frontend]     at WebpackCLI.createCompiler (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/lib/webpack-cli.js:1845:29)
[frontend]     at Command.<anonymous> (/Users/offbynone/code/work/metabase/node_modules/@webpack-cli/serve/lib/index.js:81:30)
[frontend]     at async Promise.all (index 1)
[frontend]     at Command.<anonymous> (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/lib/webpack-cli.js:1289:13)
error Command failed with exit code 2.
```

UnusedFilesWebpackPlugin is still [not compatible](https://github.com/single-spa/create-single-spa/pull/216/files/d67c16c9db23a9f98215940b060c429e62a023ea#r528952675) with the Webpack 5. Not sure will they ever update it. It seems like it's [currently stuck](https://github.com/tomchentw/unused-files-webpack-plugin/pull/41).

As a temporary solution, I'll remove it.
Possible alternative?:
- https://stackoverflow.com/questions/47913809/how-to-find-unused-files-in-a-webpack-project